cross/tcl/Makefile

```diff
@@ -2,36 +2,31 @@ PKG_NAME = tcl
 PKG_VERS = 8.6.9
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
-PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl/
+PKG_DIST_SITE = https://prdownloads.sourceforge.net/tcl/
 PKG_DIR = $(PKG_NAME)$(PKG_VERS)/unix
 
 DEPENDS =
 BUILD_DEPENDS = cross/zlib
 
 HOMEPAGE = http://www.tcl.tk/
-COMMENT  = Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
-LICENSE  = http://tcl.tk/software/tcltk/license.html
+COMMENT = Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
+LICENSE = http://tcl.tk/software/tcltk/license.html
 
 GNU_CONFIGURE = 1
 
-# Prevent re-declaration of fixstrtod.o
-# export ac_cv_func_strtod=yes
-# export tcl_cv_strtod_buggy=1
-
-CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(INSTALL_PREFIX)
+DESTDIR = /var/packages/$(PKG_NAME)/target
```

Comment:
I guess this line is now useless as standard "install" target is no longer replaced by "tcl_custom_install". Please refer to https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.install.mk#L59